app/src/main/res/layout/subscription_item.xml

```diff
@@ -37,12 +42,37 @@
         android:id="@+id/triangleCountView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true"
         android:layout_alignParentTop="true"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
         app:backgroundColor="?attr/colorSecondary"
         app:corner="rightTop"
         app:primaryText="Test"
         app:primaryTextColor="?attr/colorOnSecondary"
         app:primaryTextSize="12sp" />
+
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
+        android:id="@+id/selectView"
+        android:layout_width="55dp"
+        android:layout_height="55dp"
+        android:visibility="gone">
+
+        <ImageView
+            android:layout_width="60dp"
+            android:layout_height="60dp"
+            android:transformPivotX="0dp"
+            android:transformPivotY="0dp"
+            android:translationX="-22dp"
+            android:translationY="-22dp"
```

Comment:
Can't you handle this in the background image instead of moving views around? This looks like it could easily break in the future.

Reply:
It definitely not very elegant and robust. But that is what currently works and don't know of a better way. We should use this solution for now and create a separate PR for anybody that can improve it. Would be a good first issue for somebody with good knowledge of android UI.

Reply:
I don't like temporary solutions. They never get fixed because noone is motivated to touch working but inelegant code. Temporary solutions then pile up and make the whole code fragile.

Reply:
I'm not saying that is temporary (Well, most code is temporary in that that it can still change). I'm just saying that I don't know of a better solution.